sdk/securitycenter/Azure.ResourceManager.SecurityCenter/src/Generated/ArmSecurityCenterModelFactory.cs

```diff
@@ -1767,7 +1767,7 @@ public static ServerVulnerabilityProperties ServerVulnerabilityProperties(string
             cve ??= new List<SecurityCve>();
             vendorReferences ??= new List<VendorReference>();
 
-            return new ServerVulnerabilityProperties("ServerVulnerabilityAssessment", serverVulnerabilityType, cvss, isPatchable, cve?.ToList(), threat, publishedOn, vendorReferences?.ToList());
+            return new ServerVulnerabilityProperties(new AssessedResourceType("ServerVulnerabilityAssessment"), serverVulnerabilityType, cvss, isPatchable, cve?.ToList(), threat, publishedOn, vendorReferences?.ToList());
```

Comment:
code here is actually unchanged - the type of this parameter is `AssessedResourceType`.
Previously we are invoking the implicit operator from string, now we are invoking the ctor explicitly